deepmd/pt/train/training.py

```diff
@@ -273,8 +274,18 @@ def get_lr(lr_params):
         else:
             self.opt_type, self.opt_param = get_opt_param(training_params)
 
+        # loss_param_tmp for Hessian activation
+        loss_param_tmp = None
+        if not self.multi_task:
+            loss_param_tmp = config["loss"]
+        else:
+            loss_param_tmp = {
+                model_key: config["loss_dict"][model_key]
+                for model_key in self.model_keys
+            }
+
         # Model
-        self.model = get_model_for_wrapper(model_params)
+        self.model = get_model_for_wrapper(model_params, loss_param_tmp)
```

Comment:
I do not understand why you need the loss parameters to get a model.

Reply:
`EnergyHessianStdLoss` takes the similar form of `EnergyStdLoss`. Both of them utilize `ener` as the type of `fitting_net`. As described in the corresponding [documentation](https://github.com/1azyking/deepmd-kit/blob/enerhess/doc/model/train-energy-hessian.md), one who wants to train with Hessian can simply add the parameters `start_pref_h` and `limit_pref_h` to the `loss` section of the input file, without modifying the `type` of `fitting_net` or `loss` section. Therefore, in the function `get_model_for_wrapper`, `_loss_params` is expected to be passed to check for the valid `pref_h` parameters, and it is not possible to distinguish between `EnergyModel` and `EnergyHessianModel` solely from `_model_params`. In my view, to pass the `_loss_params` might be the simplest method to get `EnergyHessianModel`.